runtime/src/handler/map_handler.cpp

```diff
@@ -474,6 +484,7 @@ int bpf_map_handler::map_init(managed_shared_memory &memory)
 			container_name.c_str())(memory, key_size, value_size);
 		return 0;
 	}
+	#if __linux__
```

Comment:
`USE_BPF`?

Reply:
i can use that but since i have added guards for kernel maps, I preferred `__linux__` 
@Officeyutong  @yunwei37  what do you suggest?

Reply:
> i can use that but since i have added guards for kernel maps, I preferred `__linux__` @Officeyutong @yunwei37 what do you suggest?

`USE_BPF` might be better, since kernel user shared maps relies on libbpf to work

Reply:
I have replaced __linux__ with USE_LIBBPF